docs/core/tools/nuget-signed-package-verification.md

```diff
@@ -24,10 +24,13 @@ The following sections for each operating system describe:
 
 Verification is enabled by default during package restore operations.
```

Comment:
I know this is not part of your changes, but would you mind removing "by default" from this sentence to avoid giving an impression that the verification functionality can be disabled on Windows?

Reply:
@aortiz-msft, how about "enabled by default" -> "always enabled"?